src/OpenTelemetry/AssemblyInfo.cs

```diff
@@ -17,6 +17,7 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("OpenTelemetry.Tests" + AssemblyInfo.PublicKey)]
+[assembly: InternalsVisibleTo("OpenTelemetry.Exporter.Console.Tests" + AssemblyInfo.PublicKey)]
```

Comment:
What `internal` member of the ConsoleExporter do the tests need access to?

Reply:
The test was using an internal ctor for `Batch`. 
This test was originally in another project (with internals) before moving to its own project.

I made a change to use a public ctor and removed this.